packages/components/pie-button/src/index.ts

```diff
@@ -16,6 +16,11 @@ export {
 
 const componentSelector = 'pie-button';
 
+/**
+ * @slot icon-leading - Leading icon
```

Comment:
Were these comments added for a specific reason? I'm trying to figure out other scenarios where this applies.

Reply:
It allows the `custom-elements.json` file to correctly list any slots used by the component. I do not think the CEM analyser picks them up by default.